EDIT: Resolved as soon as I submitted this -- All queries fail

EDIT: This was a temporary issue and is working for me again. Feel free to close if this doesn't seem important. 

I believe this started for me sometime between March 7 16:45 EST and 17:15 EST. 

All queries, having tried on both a PC and phone, result in the loading icon running forever. The UI stays in tact and is interactive though. 
![image](https://github.com/GenSpectrum/cov-spectrum-website/assets/94097102/a91b0504-69b0-426c-a9ba-cb90ce3dad63)


![image](https://github.com/GenSpectrum/cov-spectrum-website/assets/94097102/5e0676e2-6f07-4730-a11d-67ce8f653e2b)
Failed to load resource: the server responded with a status of 503 ()
instrument.ts:124 Error: Error fetching info
    at api-lapis.ts:83:11
    at p (regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (regeneratorRuntime.js:133:17)
    at Generator.next (regeneratorRuntime.js:74:21)
    at o (asyncToGenerator.js:3:20)
    at l (asyncToGenerator.js:22:9)
(anonymous) @ instrument.ts:124
Show 1 more frame
Show less
2instrument.ts:124 DOMException: The user aborted a request.

--- a/src/data/api-lapis.ts
+++ b/src/data/api-lapis.ts
@@ -36,7 +36,11 @@ const ACCESS_KEY = process.env.REACT_APP_LAPIS_ACCESS_KEY;
 
 let currentLapisDataVersion: number | undefined = undefined;
 
-export const get = async (endpoint: string, signal?: AbortSignal) => {
+export const get = async (
+  endpoint: string,
+  signal?: AbortSignal,
+  options: { skipMaintenanceCheck?: boolean } = {}
+) => {
   let url = `${HOST}/sample${endpoint}`;
 
   const requestInit =
@@ -48,7 +52,12 @@ export const get = async (endpoint: string, signal?: AbortSignal) => {
           method: 'GET',
           signal: signal,
         };
-  return await fetch(url, requestInit);
+
+  const response = await fetch(url, requestInit);
+  if (!(options.skipMaintenanceCheck === true) && response.status === 503) {
+    window.location.reload();
+  }
+  return response;
 };
 
 export type SiloAvailability =
@@ -60,7 +69,7 @@ export async function checkSiloAvailability(signal?: AbortSignal): Promise<SiloA
   if (ACCESS_KEY) {
     url += '?accessKey=' + ACCESS_KEY;
   }
-  const response = await get(url, signal);
+  const response = await get(url, signal, { skipMaintenanceCheck: true });
 
   if (response.status !== 503) {
     return { isAvailable: true as const };
@@ -78,7 +87,7 @@ export async function fetchLapisDataVersion(signal?: AbortSignal): Promise<strin
   if (ACCESS_KEY) {
     url += '?accessKey=' + ACCESS_KEY;
   }
-  const response = await get(url, signal);
+  const response = await get(url, signal, { skipMaintenanceCheck: true });
   if (!response.ok) {
     throw new Error('Error fetching info');
   }
@@ -92,7 +101,7 @@ export async function fetchNextcladeDatasetInfo(signal?: AbortSignal): Promise<N
   if (ACCESS_KEY) {
     url += '&accessKey=' + ACCESS_KEY;
   }
-  const response = await get(url, signal);
+  const response = await get(url, signal, { skipMaintenanceCheck: true });
   if (!response.ok) {
     throw new Error('Error fetching Nextclade dataset info');
   }
@@ -108,7 +117,7 @@ export async function fetchAllHosts(): Promise<string[]> {
   if (ACCESS_KEY) {
     url += '&accessKey=' + ACCESS_KEY;
   }
-  const res = await get(url);
+  const res = await get(url, undefined, { skipMaintenanceCheck: true });
   if (!res.ok) {
     throw new Error('Error fetching new samples data');
   }

